libnetwork/sandbox.go

```diff
@@ -89,6 +89,7 @@ type resolvConfPathConfig struct {
 	dnsList              []string
 	dnsSearchList        []string
 	dnsOptionsList       []string
+	dnsNoProxy           bool
```

Comment:
This field is not like the others. It does not affect the container /etc/resolv.conf file. Move it to `containerConfig`?

Reply:
Done.